<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="cpj" name="com.yiibai.primera.testng.cases" tests="3" failures="0" timestamp="15 Aug 2017 07:57:42 GMT" time="0.026" errors="0">
  <testcase name="@BeforeClass initDriver" time="0.0" classname="com.yiibai.primera.testng.cases.Login">
    <failure type="java.lang.AssertionError" message="expected object to not be null">
      <![CDATA[java.lang.AssertionError: expected object to not be null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.assertNotNull(Assert.java:404)
	at org.testng.Assert.assertNotNull(Assert.java:389)
	at com.yiibai.primera.testng.cases.Login.initDriver(Login.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- @BeforeClass initDriver -->
  <testcase name="@AfterClass afterClass" time="0.0" classname="com.yiibai.primera.testng.cases.Login">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="@BeforeClass initDriver" time="0.0" classname="com.yiibai.primera.testng.cases.Register">
    <skipped/>
  </testcase> <!-- @BeforeClass initDriver -->
  <testcase name="@AfterClass afterClass" time="0.0" classname="com.yiibai.primera.testng.cases.Register">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="@AfterTest afterTest" time="0.0" classname="com.yiibai.primera.testng.cases.Register">
    <skipped/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase name="loginConfim" time="0.002" classname="com.yiibai.primera.testng.cases.Login">
    <skipped/>
  </testcase> <!-- loginConfim -->
  <testcase name="EmailMissKeyChars" time="0.001" classname="com.yiibai.primera.testng.cases.Register">
    <skipped/>
  </testcase> <!-- EmailMissKeyChars -->
  <testcase name="RegisterOk" time="0.001" classname="com.yiibai.primera.testng.cases.Register">
    <skipped/>
  </testcase> <!-- RegisterOk -->
</testsuite> <!-- com.yiibai.primera.testng.cases -->
